docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -35,3 +39,26 @@ Tiered Storage uses the following:
 - Data Persistence and Tiered Storage are mutually exclusive features. If both are enabled simultaneously, `InvalidConfigurationException` is thrown during the member startup.
 
 - SQL is not supported for the Tiered-Storage-backed maps. `UnsupportedOperationException` is thrown when SQL query is executed. To avoid this, use xref:query:predicate-overview.adoc[Predicate API] instead.
+
+== Supported Devices
+
+Hazelcast supports only NVMe (local) SSD devices for Tiered Storage.
+While HDDs and some network storages work with the same file abstraction that local SSDs do, and technically speaking such setups work, the access latency of these devices compared to that of the modern SSDs are so high that using Tiered Storage with such devices are not supported.
```

Comment:
Do we officially want to say we don't support non-SSD devices? Instead, should we recommend the use of SSDs and detail I/O access time considerations/impact when using network devices?

Reply:
I'd keep support, because we targeting SSDs only, and our tests cover SSDs only. We just can't limit the supported device due to the lack of the information about the device type.

Reply:
Does Danny test using ESBs?